src/components/Observations/MapLegend.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+export type MapLegendItem = {
+  borderColor: string;
+  fillColor: string;
+  label: string;
+};
+
+export type MapLegendProps = {
+  title: string;
+  items: MapLegendItem[];
+};
+
+export default function MapLegend({ title, items }: MapLegendProps): JSX.Element {
```

Comment:
Move this into the Map directory?
It will be used for observations and the plants dashboard.

Reply:
nvm, saw the next change :)
components/common is fine too 👍